### PR TITLE
Set default values if not supplied

### PR DESCRIPTION
**Pull request by johnwatson484**
https://eaflood.atlassian.net/browse/SFI-1970
https://eaflood.atlassian.net/browse/SFI-1984

### CODE REVIEW

File: test/unit/date-convert.test.js [hunk 1]
```diff
@@ -10,4 +19,9 @@ describe('convert currency', () => {
     const result = convertToDaxDate('2021-11-01')
     expect(result).toEqual('01/11/2021')
   })
+
+  test('uses current date in correct format if no date supplied', () => {
+    const result = convertToDaxDate()
+    expect(result).toMatch('01/06/2022')
+  })
```
[the remark this answers is not in the document]
Comment (AbidemiAdio) [reply test/unit/date-convert.test.js:26]: Great, QED

**Review by AbidemiAdio — APPROVED** [review]
QED

File: test/unit/date-convert.test.js [hunk 2]
```diff
@@ -10,4 +19,9 @@ describe('convert currency', () => {
     const result = convertToDaxDate('2021-11-01')
     expect(result).toEqual('01/11/2021')
   })
+
+  test('uses current date in correct format if no date supplied', () => {
```
Comment (marctemp) [inline test/unit/date-convert.test.js:23]: Could you add other tests to cover other scenarios and data types to give us confidence of defence in depth for future integrations:

- '2021/11/01'
- '01-11-2021'
- '01-11-2021'
- 'Monday 1st November 2021'
- '32/11/2021'
- undefined
- null
- true
- false
- {}
- []
- 1
- 0
- Boolean()

Comment (johnwatson484) [reply test/unit/date-convert.test.js:23]: Tests added, I've also updated to handle different date formats.  The allowed date formats have also been updated in `asyncapi` spec.

**Pull request merged**